content/docs/glossary/multisig.mdx

```diff
@@ -55,6 +55,10 @@ If the weight of the master key is updated to 0, it cannot be used to sign trans
 
 Each additional signer beyond the master key increases the account's [minimum balance](./minimum-balance.mdx).
 
+## One-Time Extra Signers
+
+You can require that a transaction have a particular signer that isn't associated with any of the accounts used in the transaction by including it in the [transaction validity precondition](./transactions.mdx#extra-signers).
```

Comment:
Suggest using ubiquitous language for the title, and noting the one-time quality in the body.
```suggestion
## Extra Signers

You can require that a transaction have a particular signer that isn't associated with any of the accounts used in the transaction by including it in the [transaction validity precondition](./transactions.mdx#extra-signers). The extra signer applies only to that one transaction.
```